### PR TITLE
Sonarqube integration

### PR DESCRIPTION
**Pull request by TimoBbz**

### CODE REVIEW

**Comment by TimoBbz** [comment]
The issue came from a typo that prevented the end of the command (with the options) to be read, and the GitHub token was lacking. Now it does run: github.com/riseclipse/riseclipse-main/runs/6008903243
The result of the corresponding analysis can be found here: sonarcloud.io/summary/new_code?id=riseclipse_riseclipse-main&branch=sonarqube-integration

File: .github/workflows/Shared-Verify-Components.yml [hunk 1]
```diff
@@ -46,4 +49,13 @@ jobs:
...
-        mvn verify
\ No newline at end of file
+        mvn verify
+
+    - name: Run analysis with Sonarqube
+      run: |
+        cd ${{ github.event.repository.name }}/${{ inputs.build-path }}
+        mvn verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Dsonar.organization=riseclipse \
+        -Dsonar.host.url=http://sonarcloud.io -Dsonar.login=${{ secrets.SONAR_TOKEN }} \
+        -Dsonar.projectKey=riseclipse_${{ github.event.repository.name }}
```
Comment (marcadetd) [inline .github/workflows/Shared-Verify-Components.yml:59]: Indent these two lines

Comment (marcadetd) [reply .github/workflows/Shared-Verify-Components.yml]: and tools workflow?

File: .github/workflows/Shared-Verify-Components.yml [hunk 2]
```diff
@@ -57,3 +57,5 @@ jobs:
         mvn verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Dsonar.organization=riseclipse \
         -Dsonar.host.url=http://sonarcloud.io -Dsonar.login=${{ secrets.SONAR_TOKEN }} \
         -Dsonar.projectKey=riseclipse_${{ github.event.repository.name }}
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (marcadetd) [inline .github/workflows/Shared-Verify-Components.yml:61]: Why is-it needed?

Comment (TimoBbz) [reply .github/workflows/Shared-Verify-Components.yml:61]: It doesn’t seem to work without, see this action run: https://github.com/riseclipse/riseclipse-main/runs/6008778025

Comment (marcadetd) [reply .github/workflows/Shared-Verify-Components.yml:61]: Then it must be added also in the tool workflow

Comment (marcadetd) [reply .github/workflows/Shared-Verify-Components.yml:61]: Done

**Review by marcadetd — APPROVED** [review]

**Pull request merged**